docs/user-guide/troubleshooting.md

```diff
@@ -7,31 +7,109 @@ UI in their web browser and see a message like this:
 
     Get https://1.2.3.4/api/v1/replicationcontrollers: x509: failed to load system roots and no roots provided
 
-This means that the dashboard failed to authenticate to the API server. Before
+This means that Dashboard failed to authenticate to the API server. Before
 explaining the solution, it is useful to review how the dashboard discovers
 and authenticates with the API server.
 
-The dashboard uses a service account (as opposed to a user account) to communicate
-with the API server. In order to contact the API server, the dashboard expects
-the following:
+Dashboard can connect with the API server in two different ways:
 
-1. Two environment variables, `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT`
-provide the host and port of the API server. You can override these environment
-variables by providing the --apiserver-host argument when starting the dashboard
-pod, but you normally do not need to do this.
+1. Dashboard uses a service account (as opposed to a user account) to communicate
+with the API server. In most cases this is the easiest way to get Dashboard
+running as no configuration is required.
 
-2. A file, `/var/run/secrets/kubernetes.io/serviceaccount/token` provides a secret
+2. The Dashboard binary can be started with the `--kubeconfig` flag.
+The value of the flag is a path to a file specifying how to connect to the API server.
+The contents of the file is identical to `~/.kube/config` which is used by kubectl to
+connect to the API server.
+
+## Service Accounts
+Dashboard expects the file
```

Comment:
"If using a service account to connect to the API server ..."

I'm assuming this isn't relevant if using the --kubeconfig flag.